lib/fs.js

```diff
@@ -1635,7 +1635,33 @@ function stat(path, options = { bigint: false }, callback) {
   callback = makeStatsCallback(callback);
 
   const req = new FSReqCallback(options.bigint);
-  req.oncomplete = callback;
+
+  options = getOptions(options, { bigint: false, signal: undefined });
```

Comment:
JSDoc and default value of `options` should be updated accordingly.

Reply:
I wonder what version I should enter when adding the new feature, would you have any suggestions?

  ```
    - Version: 
    pr-url: https://github.com/nodejs/node/pull/57775
    Description: Now accepts an additional `signal` property to allow the transaction to be canceled.
```

Reply:
Use `REPLACEME`, it will be updated with actual version(s) when commit gets into release.
Refs: https://github.com/nodejs/node/blob/main/doc/contributing/pull-requests.md#step-3-code

Reply:
I used, thanks